server/src/main/java/org/elasticsearch/script/ScoreScript.java

```diff
@@ -34,7 +35,7 @@
 /**
  * A script used for adjusting the score on a per document basis.
  */
-public abstract class ScoreScript {
+public abstract class ScoreScript implements ScorerAware {
```

Comment:
What's the reason for needing to add ScorerAware?

Reply:
@rjernst Thanks Ryan, I was copying this from a former SearchScript, this is removed now